nginx.tmpl.master

```diff
@@ -507,9 +497,9 @@ server {
         {{ end }}
     {{ end }}
 
-    {{- if and (exists "/etc/nginx/conf_custom.d/ip-whitelist.conf") ($include) }}
-    include /etc/nginx/conf_custom.d/ip-whitelist.conf;
-    {{- end }}
+    #{{- if and (exists "/etc/nginx/conf_custom.d/ip-whitelist.conf") ($include) }}
+    #include /etc/nginx/conf_custom.d/ip-whitelist.conf;
+    #{{- end }}
```

Comment:
Is this left commented out on purpose or can it be deleted?
Otherwise, looks good to me.